pkg/rancher-desktop/main/commandServer/settingsValidator.ts

```diff
@@ -492,6 +492,79 @@ export default class SettingsValidator {
     return false;
   }
 
+  protected checkExtensions(
```

Comment:
Could you add some unit tests in `pkg/rancher-desktop/main/commandServer/__tests__/settingsValidator.spec.ts` on this?